### PR TITLE
Release 1.100.1

### PR DESCRIPTION
**Pull request by derekblank**
Release for Gutenberg Mobile 1.100.1

## Related PRs

- Gutenberg: https://github.com/WordPress/gutenberg/pull/52818
- WPAndroid: https://github.com/wordpress-mobile/WordPress-Android/pull/18808
- WPiOS: https://github.com/wordpress-mobile/WordPress-iOS/pull/21159

## Extra PRs that Landed After the Release Was Cut

No extra PRs yet. 🎉

## Changes in `1.100.0`

Note that changes from `1.100.0` are included in this PR, as release PR https://github.com/wordpress-mobile/gutenberg-mobile/pull/5981 will be closed in favor of this one.

### Change 1: Move undo/redo buttons to header toolbar
- **PR:** https://github.com/wordpress-mobile/gutenberg-mobile/pull/5889
- **Issue:** [Move undo/redo to header](https://github.com/orgs/wordpress-mobile/projects/193/views/1?pane=issue&itemId=31282466)

### Change 2: Update blue borders around blocks
- **PR 1:** https://github.com/wordpress-mobile/gutenberg-mobile/pull/5833
- **PR 2:** https://github.com/wordpress-mobile/gutenberg-mobile/pull/5969
- **Issue:** [Update blue border around blocks](https://github.com/orgs/wordpress-mobile/projects/193/views/1?pane=issue&itemId=30514957)

### Change 3: Style and Color Updates
- **PR 1:** https://github.com/wordpress-mobile/gutenberg-mobile/pull/5941
- **PR 2:** https://github.com/wordpress-mobile/gutenberg-mobile/pull/5940
- **Issue:** [Style and color updates](https://github.com/orgs/wordpress-mobile/projects/193/views/1?pane=issue&itemId=30514931)

### Change 4: Update toolbar icons and colors
- **PR:** https://github.com/wordpress-mobile/gutenberg-mobile/pull/5940

### Change 5: Expand analytics for block insertion/replacement
- **PR:** https://github.com/wordpress-mobile/gutenberg-mobile/pull/5976

### Change 6: Add media inserter blocks to toolbar
- **PR:** https://github.com/WordPress/gutenberg/pull/51827

### Change 1:  E2E tests - Update hiding of the keyboard for iPad
- **PR:** https://github.com/WordPress/gutenberg/pull/52771

### Change 2: Fix crash when registering non-core blocks
- **PR:** https://github.com/wordpress-mobile/gutenberg-mobile/pull/5982

## Test plan

Once the installable builds of the main apps are ready, perform a quick smoke test of the editor on both iOS and Android to verify it launches without crashing. We will perform additional testing after the main apps cut their releases.

## Release Submission Checklist

- [ ] Verify Items from test plan have been completed
- [x] Check if `RELEASE-NOTES.txt` is updated with all the changes that made it to the release. Replace `Unreleased` section with the release version and create a new `Unreleased` section.
- [x] Check if `gutenberg/packages/react-native-editor/CHANGELOG.md` is updated with all the changes that made it to the release. Replace `## Unreleased` with the release version and create a new `## Unreleased`.
- [x] Bundle package of the release is updated.

### CODE REVIEW

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -1,11 +1,15 @@
 Unreleased
 ---
+
+1.100.1
+---
```
Comment (fluiddot) [inline RELEASE-NOTES.txt:5]: For consistency with versions, I'm wondering if we should have an entry for `1.100.0` with the changes made in that version 🤔. I.e.

```
1.100.1
---
* [**] Fix crash when registering non-core blocks [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5982]

1.100.0
---
* [**] Add media inserter buttons to editor toolbar [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5900]
* [**] Update native BlockOutline component styles to remove blue border from blocks [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5833]
* [**] Move the undo/redo buttons to the navigation bar [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5889]
* [**] Update Editor block inserter button styles and default text input placeholder/selection styles [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5941]
* [**] Update Editor toolbar icons and colors [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5940]
* [**] Update media placeholders and block outline [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5969]
```

Comment (fluiddot) [reply RELEASE-NOTES.txt:5]: I applied this suggestion via https://github.com/wordpress-mobile/gutenberg-mobile/pull/5985/commits/92c3486f902176163135eb5c9ac671fc0d43eae1.

**Review by fluiddot — APPROVED** [review]
LGTM 🎊 !

I confirmed this PR was branched off from a `trunk`'s commit that includes the fixes:
- https://github.com/wordpress-mobile/gutenberg-mobile/pull/5982

**Comment by derekblank** [comment]
## 🟢   Writing Flow (iOS)
**Build**: 22.9.0.0 (TestFlight)
**Device**: iPhone SE (iOS 16.5.1)

#### General
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/copy-paste.md#tc001) - Paste formatted text copied from website
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/multiline-components.md#tc001) - Multiline components
  - [x] Quote block
  - [x] Verse block
  - [x] Preformatted block
  - [ ] Code block (DEV only)
  - [x] Pullquote block
#### Rich Text Format
- [x] [TC007](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc007) - Test format detection under the cursor
- [x] [TC009](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc009) - Test autocorrection doesn't apply formatting to Heading
- [x] [TC010](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc010) - Test autocorrection doesn't remove formatting from Heading
##### Splitting and merging
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc001) - Merge after writing
  - [x] Paragraph
  - [x] Heading
- [x] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc002) - Merge after selection
  - [x] Paragraph
  - [x] Heading
- [x] [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc003) - Merge after deleting text
  - [x] Paragraph
  - [x] Heading
- [x] [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc004) - Merge after deleting all
  - [x] Paragraph
  - [x] Heading
- [x] [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc005) - Merge multiple blocks
  - [x] Paragraph
  - [x] Heading

**Comment by derekblank** [comment]
## 🟢  Unsupported Block Tests (iOS)
**Build**: 22.9.0.0 (TestFlight)
**Device**: iPhone SE (iOS 16.5.1)

- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc001) - User can edit unsupported blocks on Simple WP.com sites
- [x] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc002) - User can discard edits to an unsupported block on Simple WP.com sites
- [x] [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc003) - Editing unsupported blocks is allowed on Gutenberg-enabled Atomic sites
- [x] [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc004) - Editing unsupported blocks is disallowed on Classic-enabled Atomic sites
- [x] [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc005) - Editing unsupported blocks is enabled on self-hosted sites accessed via Jetpack
- [x] [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc006) - Editing unsupported blocks is disallowed on self-hosted sites access via their own username and password

**Comment by derekblank** [comment]
## 🟢  Release Changes (iOS)
**Build**: 22.9.0.0 (TestFlight)
**Device**: iPhone SE (iOS 16.5.1)

- [x] [Move undo/redo to header](https://github.com/orgs/wordpress-mobile/projects/193/views/1?pane=issue&itemId=31282466)
- [x] [Updated blue border around blocks](https://github.com/WordPress/gutenberg/pull/52702)
- [x] [Updated toolbar icons and colors](https://github.com/WordPress/gutenberg/pull/52336)
- [x] [Fix crash when registering non-core blocks](https://github.com/wordpress-mobile/gutenberg-mobile/pull/5982)
- [x] [Add media inserter blocks to toolbar](https://github.com/WordPress/gutenberg/pull/51827)
- [x] [Update Editor block inserter button styles and default text input placeholder/selection styles](https://github.com/WordPress/gutenberg/pull/52269)

**Comment by derekblank** [comment]
## 🟢   Functionality Test Suite 1 (iOS)
**Build**: 22.9.0.0 (TestFlight)
**Device**: iPhone SE (iOS 16.5.1)

### Gallery - 1

- [x] Gallery block - Close/Re-open post with an ongoing image upload - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/gallery.md#tc001)
- [x] Gallery block - Close post with an ongoing image upload - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/gallery.md#tc002)

### Gallery - 2

- [x] Gallery block - Insert image from device (failing) - [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/gallery.md#tc006)
- [x] Gallery block - Insert image from device (cancel) - [TC009](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/gallery.md#tc009)
- [x] Gallery block - Try adding same images from WP Media library and moving the images around - [TC010](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/gallery.md#tc010)

### MediaText - 1

- [x] Media Text block - Insert image from device (failing) - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/media-text.md#tc001-i)
- [x] Media Text block - Close/Re-open post with an ongoing image upload - [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/media-text.md#tc003-i)

### MediaText - 2

- [x] Media Text block - Insert video from device - [TC002-v](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/media-text.md#tc002-v)
- [x] Media Text block - Insert image from device - [TC002-i](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/media-text.md#tc002-i)

### MultipleUpload - 1

- [x] Simultaneous uploads - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/media-interaction.md#tc001)

### Cover - 1

- [x] Cover - Add image background from WordPress Media library - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/cover.md#tc001)
- [x] Cover - Image focal point is rendered properly - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/cover.md#tc002)
- [x] Cover - Custom color is rendered properly - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/cover.md#tc003)

### Editor Theme - 1

- [x] Default Colors - Check that default colors still load - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/editor-theme.md#tc001)
- [x] Default Gradients - Check that default gradients still load - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/editor-theme.md#tc002)
- [x] Custom Colors - Check that custom colors load in the editor - [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/editor-theme.md#tc003)

### Social Icons - 1
- [x] Active icon gets product color - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/social-icons.md#tc001)
- [x] Social icon forwarding to the link - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/social-icons.md#tc002)

**Comment by derekblank** [comment]
## 🟢 Writing Flow (Android)
**Build**: jpandroid-22.9-rc-1
**Device**: Samsung Galaxy S20 FE (Android 12)

#### General
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/copy-paste.md#tc001) - Paste formatted text copied from website
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/multiline-components.md#tc001) - Multiline components
  - [x] Quote block
  - [x] Verse block
  - [x] Preformatted block
  - [ ] Code block (DEV only)
  - [x] Pullquote block
#### Rich Text Format
- [x] [TC007](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc007) - Test format detection under the cursor
- [x] [TC009](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc009) - Test autocorrection doesn't apply formatting to Heading
- [x] [TC010](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc010) - Test autocorrection doesn't remove formatting from Heading
##### Splitting and merging
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc001) - Merge after writing
  - [x] Paragraph
  - [x] Heading
- [x] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc002) - Merge after selection
  - [x] Paragraph
  - [x] Heading
- [x] [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc003) - Merge after deleting text
  - [x] Paragraph
  - [x] Heading
- [x] [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc004) - Merge after deleting all
  - [x] Paragraph
  - [x] Heading
- [x] [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc005) - Merge multiple blocks
  - [x] Paragraph
  - [x] Heading
#### Undo / Redo - Test Cases
- [ ] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/undo-redo.md#tc002) - Undo/redo text

**Comment by derekblank** [comment]
## 🟢    Unsupported Block Tests (Android)
**Build**: jpandroid-22.9-rc-1
**Device**: Samsung Galaxy S20 FE (Android 12)

- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc001) - User can edit unsupported blocks on Simple WP.com sites
- [x] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc002) - User can discard edits to an unsupported block on Simple WP.com sites
- [x] [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc003) - Editing unsupported blocks is allowed on Gutenberg-enabled Atomic sites
- [x] [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc004) - Editing unsupported blocks is disallowed on Classic-enabled Atomic sites
- [x] [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc005) - Editing unsupported blocks is enabled on self-hosted sites accessed via Jetpack
- [x] [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc006) - Editing unsupported blocks is disallowed on self-hosted sites access via their own username and password

**Comment by derekblank** [comment]
## 🟢      Functionality Test Suite 5 (Android)
**Build**: jpandroid-22.9-rc-1
**Device**: Samsung Galaxy S20 FE (Android 12)

Video block - 1
- [x] Video block - Insert video from device (failing) - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/video.md#tc001)
- [x] Video block - Close/Re-open post with an ongoing video upload - [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/video.md#tc004)

Video block - 2
- [x] Video block - Insert video from device (cancel) - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/video.md#tc002)
- [x] Video block - Close post with an ongoing video upload - [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/video.md#tc005)
- [x] Video block - Add Caption - [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/video.md#tc003)

File block - 1
- [x] File block - Upload completes successfully from the device library - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc001)
- [x] File block - Insert completes successfully from the WordPress Media library - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc002)
- [x] File block - Upload fails from the device library - [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc003)
- [x] File block - Stop the file upload before it completes - [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc004)

File block - 2
- [x] File block - Close the post with an ongoing file upload - [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc005)
- [x] File block - Replace the selected files - [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc006)
- [x] File block - Change the alignment in the File Block - [TC007](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc007)
- [x] File block - The Link To Block Setting should open - [TC008](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc008)

File block - 3
- [x] File block - Open link in new tab - [TC009](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc009)
- [x] File block - Show/hide download button (Block Settings only) - [TC010](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc010)
- [x] File block - Copy file URL - [TC011](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc011)
- [x] File block - Close/Re-open post with an ongoing file upload - [TC012](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc012)

Audio block - 1
- [x] Audio block - Upload completes successfully from the device library - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc001)
- [x] Audio block - Insert completes successfully from the WordPress Media library - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc002)

Audio block - 2
- [x] Audio block - Upload fails from the device library - [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc003)
- [x] Audio block - Stop the audio file upload before it completes - [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc004)

Audio block - 3
- [x] Audio block - Close the post with an ongoing audio file upload - [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc005)
- [x] Audio block - Replace the selected files - [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc006)
- [x] Audio block - Change the alignment in the Audio Block - [TC007](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc007)

Audio block - 4
- [x] Audio block - Autoplay setting - [TC008](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc008)
- [x] Audio block - Loop setting - [TC009](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc009)
- [x] Audio block - Close/Re-open post with an ongoing audio file upload - [TC010](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc010)

Synced patterns - 1
- [x] Synced patterns - Add pattern - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/synced-patterns.md#tc001)
- [x] Synced patterns - Detach pattern - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/synced-patterns.md#tc002)

Synced patterns - 2
- [x] Synced patterns - Deleted pattern - [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/synced-patterns.md#tc003)
- [x] Synced patterns - Recursive block rendering - [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/synced-patterns.md#tc004)

Embed block - 1
- [x] Embed block - Inline previews are rendered correctly for the top 5 most-used providers - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/embed.md#tc001)

Embed block - 2
- [x] Embed block - Inline previews are rendered accordingly for each alignment option - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/embed.md#tc002)

**Comment by derekblank** [comment]
## 🟢  Release Changes (Android)
**Build**: jpandroid-22.9-rc-1
**Device**: Samsung Galaxy S20 FE (Android 12)

- [x] [Move undo/redo to header](https://github.com/orgs/wordpress-mobile/projects/193/views/1?pane=issue&itemId=31282466)
- [x] [Updated blue border around blocks](https://github.com/WordPress/gutenberg/pull/52702)
- [x] [Updated toolbar icons and colors](https://github.com/WordPress/gutenberg/pull/52336)
- [x] [Fix crash when registering non-core blocks](https://github.com/wordpress-mobile/gutenberg-mobile/pull/5982)
- [x] [Add media inserter blocks to toolbar](https://github.com/WordPress/gutenberg/pull/51827)
- [x] [Update Editor block inserter button styles and default text input placeholder/selection styles](https://github.com/WordPress/gutenberg/pull/52269)

**Pull request merged**